.github/workflows/nightly.yaml

```diff
@@ -9,9 +9,19 @@ on:
   schedule:
   # 05:00 UTC = 06:00 CET = 07:00 CEST
   - cron: "0 5 * * *"
+  pull_request_target:
+    paths:
+      - '**.gms'
+      - '**.gpr'
+      - '**.gdx'
+    types:
+     - labeled
```

Comment:
Without this, our secrets could be extracted by any workflow changing GAMS files.

Reply:
Note: the tests passed before, so I disabled them again. They won't run when we apply a label to this PR because it would need to clear both filters, i.e. change some GAMS files *and* get a label to run.